client/src/main/java/gui/controllers/PlayerSceneController.java

```diff
@@ -36,8 +37,8 @@ public void initialize() {
     @FXML
     private void startPlacing() throws Exception {
         Window currentWindow = startPlacement.getScene().getWindow();
-        savePlayer();
         tryToConnect();
+        savePlayer();
```

Comment:
savePlayer() should not be invoked if tryToConnect() failed

Reply:
Will be in the future. This issue is tracked in #33